.travis.yml

```diff
@@ -16,7 +16,6 @@ install:
   # Install test requirements
   - pip install -e ".[test]"
   # Now install it as a standalone distribution
-  - pip uninstall -y .
```

Comment:
This should not have been removed. The point of this line is to uninstall the development version (i.e. `-e` option to pip). This smoke tests the package manifest is accurately includes the files that need to be available in the packaged version.